draft-ietf-mls-protocol.md

```diff
@@ -853,14 +855,16 @@ update_secret -> HKDF-Extract = epoch_secret
                      +--> Derive-Secret(., "app", GroupState_[n])
                      |    = application_secret
                      |
+                     +--> Derive-Secret(., "confirm", GroupState_[n])
+                     |    = key_confirmation
```

Comment:
Can't we send the next `init_secret`instead ?

Reply:
That seems bad, since the `init_secret` feeds into the next `epoch_secret`.  It does get mixed with the `update_secret` there, but it would still would make me even more worried than this proposal does.

Reply:
Indeed it is preferable to avoid it. I was just wondering if we can reuse something we already have...